rest-api/test/unit_test/api_test/biobank_order_api_test.py

```diff
@@ -65,8 +65,12 @@ def test_auto_pair_updates_participant_and_summary(self):
     p_paired = self.participant_dao.get(self.participant.participantId)
     self.assertNotEqual(p_paired.hpoId, UNSET_HPO_ID)
     self.assertIsNotNone(p_paired.providerLink)
+
     s_paired = self.summary_dao.get(self.participant.participantId)
+
     self.assertNotEqual(s_paired.hpoId, UNSET_HPO_ID)
+    self.assertEqual(s_paired.biospecimenFinalizedSiteId, s_paired.siteId)
+    self.assertEqual(s_paired.physicalMeasurementsFinalizedSiteId, None)
```

Comment:
I don't understand this test... there's no reason to think this field would have been set.

Reply:
removed.